### DISCUSSION TITLE
TheXTech's Content (NPCs (without Misc. section))

### DISCUSSION THREAD

**Discussion by Core-LOVE** [Wohlstand/TheXTech#475] [Planning]
This is mostly a port of planned content from SMR, but anyways.
| ID  | Name                              | Tileset    |
|-----|-----------------------------------|------------|
| 293 | Starman                        | SMB1       |
| 294 | Bill Blaster                      | SMB1       |
| 295 | Bullet Bill                       | SMB1       |
| 296 | Flagpole                          | SMB1       |
| 297 | Red Spring                        | SMB1       |
| 298 | Green Spring                      | SMB1 (TLL) |
| 299 | Lakitu                            | SMB1       |
| 300 | Spiny                             | SMB1       |
| 301 | Spiny Egg                         | SMB1       |
| 302 | Balance Platforms                 | SMB1       |
| 303 | Balance Platform                  | SMB1       |
| 304 | Fire-Bar (New)                    | SMB1       |
| 305 | Piranha Plant                     | SMB1       |
| 306 | Red Piranha Plant                 | SMB1 (TLL) |
| 307 | Pink Piranha Plant                | SMB1 (MF)  |
| 308 | Purple Piranha Plant              | SMB1 (MF)  |
| 309 | Piranha Plant (Horizontal)        | SMB1       |
| 310 | Red Piranha Plant (Horizontal)    | SMB1 (TLL) |
| 311 | Pink Piranha Plant (Horizontal)   | SMB1 (MF)  |
| 312 | Purple Piranha Plant (Horizontal) | SMB1 (MF)  |
| 313 | Fireball (Enemy)                  | SMB1       |
| 314 | Albatoss                          | SMB2       |
| 315 | Autobomb                          | SMB2       |
| 316 | Autobomb's Projectile             | SMB2       |
| 317 | Flurry                            | SMB2       |
| 318 | Ostro                             | SMB2       |
| 319 | Pidgit                            | SMB2       |
| 320 | Flying Carpet                     | SMB2       |
| 321 | Porcupo                           | SMB2       |
| 322 | Trouter                           | SMB2       |
| 323 | Red Panser                        | SMB2       |
| 324 | Green Panser                      | SMB2       |
| 325 | Blue Panser                       | SMB2       |
| 326 | Fireball (Enemy)                  | SMB2       |
| 327 | Phanto                            | SMB2       |
| 328 | Tryclyde                          | SMB2       |
| 329 | Red Birdo                         | SMB2       |
| 330 | Green Birdo                       | SMB2       |
| 331 | Robirdo                           | SMB2 (SMA) |
| 332 | Robirdo's Egg                     | SMB2 (SMA) |
| 333 | Mask Gate                         | SMB2       |
| 334 | Clawgrip                          | SMB2       |
| 335 | Rock                              | SMB2       |
| 336 | Platform (Waterfall)              | SMB2       |
| 337 | Turtle Shell                      | SMB2           |
| 338 | Small heart                       | SMB2           |
| 339 | Starman                           | SMB2           |
| 340 | Rocket                            | SMB2           |
| 340 | Red Beezo                         | SMB2           |
| 341 | Yellow Beezo                      | SMB2           |
| 342 | Cobrat                            | SMB2           |
| 343 | Cobrat (Hiding)                   | SMB2           |
| 344 | Fryguy                            | SMB2           |
| 345 | Mask Gate (Boss)                  | SMB2           |
| 346 | Cherry                            | SMB2           |
| 347 | Mask Gate (Boss)                  | SMB2           |
| 348 | Mushroom Block (Random)           | SMB2           |
| 349 | Giant Vegetable                   | SMB2 (SMA)     |
| 350 | Giant POW                         | SMB2 (SMA)     |
| 351 | Whale                             | SMB2           |
| 352 | Waterspout                        | SMB2           |
| 353 | Big Shy Guy (Red)                 | SMB2 (SMA)     |
| 354 | Big Shy Guy (Blue)                | SMB2 (SMA)     |
| 355 | Heart                             | SMB2 (SMA)     |
| 356 | Ninji (Running)                   | SMB2        |
| 357 | Big Ninji                         | SMB2 (SMA)        |
| 358 | Green Snifit                      | SMB2        |
| 359 | Purple Snifit                     | SMB2        |
| 360 | Carrot Platform                   | SMB2 (SMA)        |
| 361 | Bouncing blade                    | SMB2 (SMA)        |
| 362 | Ferris Wheel                      | SMB2 (SMA)        |
| 363 | Ferris Wheel's Platform           | SMB2 (SMA)        |
| 364 | Spark Chaser                      | SMB2 (SMA)        |
| 365 | Bubble                            | SMB2 (SMA)        |
| 366 | Magic Potion (New)                | SMB2        |
| 367 | Magic Potion's Door               | SMB2        |
| 368 | Grass (New)                       | SMB2        |
| 369 | Toad (Friendly)                   | SMB2        |
| 370 | Peach (Friendly)                  | SMB2        |
| 371 | Chain Chomp Post                  | SMB3        |
| 372 | Chain Chomp                       | SMB3        |
| 373 | Fiery Walking Piranha             | SMB3        |
| 374 | Fire Brother                      | SMB3        |
| 375 | Fire Chomp                        | SMB3        |
| 376 | Firesnake                         | SMB3        |
| 377 | Firesnake (Tail)                  | SMB3        |
| 378 | Hot Foot                          | SMB3        |
| 379 | Hammer Brother                    | SMB3        |
| 380 | Hammer                            | SMB3        |
| 381 | Boomerang Brother                 | SMB3        |
| 382 | Boomerang                         | SMB3        |
| 383 | Sledge Brother                    | SMB3        |
| 384 | Thwomp's Shoe                     | SMB3        |
| 385 | Kuribo's Shoe Goomba              | SMB3        |
| 386 | Podoboo's Shoe Goomba             | SMB3        |
| 387 | Lakitu's Shoe Goomba              | SMB3        |
| 388 | Thwomp's Shoe Goomba              | SMB3        |
| 389 | Bowser Statue (Laser)             | SMB3        |
| 390 | Laser                             | SMB3        |
| 390 | Buster Beetle                     | SMB3        |
| 391 | Blooper Nanny                     | SMB3        |
| 392 | Blooper Baby                      | SMB3        |
| 393 | Ptooie                            | SMB3        |
| 394 | Ptooie's Ball                     | SMB3        |
| 395 | Rocky Wrench                      | SMB3        |
| 396 | Wrench                            | SMB3        |
| 396 | Spike                             | SMB3        |
| 396 | Spike's Ball                      | SMB3        |
| 397 | Angry Sun                         | SMB3        |
| 398 | Spiny Cheep-Cheep                 | SMB3        |
| 399 | Boss Bass                         | SMB3        |
| 400 | Baby Cheep                        | SMB3        |
| 401 | Lava Lotus                        | SMB3        |
| 402 | Lava Lotus's Fireball             | SMB3        |
| 403 | Piranha Plant                     | SMB3        |
| 404 | Green Piranha Plant               | SMB3        |
| 405 | Fiery Piranha Plant               | SMB3        |
| 406 | Piranhacus Giganticus             | SMB3        |
| 407 | Piranha Plant (Horizontal)        | SMB3        |
| 408 | Green Piranha Plant (Horizontal)  | SMB3        |
| 409 | Fiery Piranha Plant (Horizontal)  | SMB3        |
| 410 | Piranhacus Giganticus (Horizontal)| SMB3        |
| 411 | Venus Fire Trap                   | SMB3        |
| 412 | Venus Fire Trap (Horizontal)      | SMB3        |
| 413 | Pipe Ptooie                       | SMB3        |
| 414 | Roto-Disc (New)                   | SMB3        |
| 415 | Boom Boom (Winged)                | SMB3        |
| 418 | Wendy O. Koopa                    | SMB3        |
| 419 | Wendy O. Koopa (Shell)            | SMB3        |
| 420 | Candy Ring                        | SMB3        |
| 421 | Iggy Koopa                        | SMB3        |
| 422 | Iggy Koopa (Shell)                | SMB3        |
| 423 | Roy Koopa                         | SMB3        |
| 424 | Roy Koopa (Shell)                 | SMB3        |
| 425 | Lemmy Koopa                       | SMB3        |
| 426 | Lemmy Koopa (Shell)               | SMB3        |
| 427 | Magical Ball                      | SMB3        |
| 428 | Magic Wand                        | SMB3        |
| 429 | Frog Suit                         | SMB3        |
| 430 | P-Wing                            | SMB3        |
| 431 | Anchor (Overworld)                | SMB3        |
| 432 | Hammer (Overworld)                | SMB3        |
| 433 | Jugem's Cloud (Overworld)         | SMB3        |
| 434 | Magic Whistle (Overworld)         | SMB3        |
| 435 | Music Box (Overworld)             | SMB3        |
| 436 | Chest                             | SMB3        |
| 437 | Ground Cannon                     | SMB3        |
| 438 | Cannonball                        | SMB3        |
| 439 | Cannon                            | SMB3        |
| 440 | Cannon Bob-Omb                    | SMB3        |
| 441 | Giant cannonball                  | SMB3        |
| 442 | Burner                            | SMB3        |
| 443 | Burner (Fire)                     | SMB3        |
| 444 | Para-Beetle                       | SMB3        |
| 445 | Donut Block (Solid)               | SMB3        |
| 446 | Red Donut Block (Solid)           | SMB3        |
| 447 | Arrow Lift                        | SMB3        |
| 448 | Arrow Lift's Ghost                | SMB3        |
| 449 | Missile Bill                      | SMB3        |
| 450 | Rotary Lift                       | SMB3        |
| 451 | Seesaw                            | SMB3 (SMA4) |
| 452 | Spiked Sphere                     | SMB3 (SMA4) |
| 453 | Spiked Sphere (Yellow)            | SMB3 (SMA4) |
| 454 | Spiked Sphere (Blue)              | SMB3 (SMA4) |
| 455 | Mini Goomba                       | SMB3        |
| 456 | Para-Goomba Nanny                 | SMB3        |
| 457 | Pile Driver Micro-Goomba          | SMB3        |
| 458 | Stretch                           | SMB3        |
| 459 | Bomb                              | SMB3 (SMA4) |
| 460 | Beach Ball                        | SMB3 (SMA4) |
| 461 | Number Block                      | SMB3 (SMA4) |
| 462 | Lakitu's Cloud                    | SMB3 (SMA4) |
| 463 | Prize Ball                        | SMB3 (SMA4) |
| 464 | Boomerang (Player)                | SMB3 (SMA4) |
| 465 | Bolt Lift                         | SMB3        |
| 466 | Orange Note Block                 | SMB3        |
| 467 | Dino Rhino                        | SMW         |
| 468 | Dino Torch                        | SMW         |
| 469 | Dino Torch's Fire                 | SMW         |
| 470 | Dino Torch's Fire (Horizontal)    | SMW         |
| 471 | Thwimp                            | SMW         |
| 472 | Minigame Coin                     | SMW         |
| 473 | Minigame Cloud                    | SMW         |
| 474 | Big Boo (New)                     | SMW         |
| 475 | Fishin' Boo                       | SMW         |
| 476 | Blue Fire                         | SMW         |
| 477 | Thwomp                            | SMW         |
| 478 | Thwomp (Horizontal)               | SMW         |
| 479 | Magikoopa                         | SMW         |
| 480 | Magikoopa's Magic                 | SMW         |
| 481 | Hidden Item                       | SMW         |
| 482 | Saw                               | SMW         |
| 483 | Saw (Vertical)                    | SMW         |
| 484 | Engine Block                      | SMW         |
| 485 | Rope                              | SMW         |
| 486 | Snake Block                       | SMW         |
| 487 | Coin Snake                        | SMW         |
| 488 | Countdown Platform                | SMW         |
| 489 | Gray Platform                     | SMW         |
| 490 | Platform                          | SMW         |
| 491 | Cannon                            | SMW         |
| 492 | Cannon (Horizontal)               | SMW         |
| 493 | Grinder Platform                  | SMW         |
| 494 | Flying Spiny                      | SMW         |
| 495 | Spiny's Needle                    | SMW         |
| 496 | Flame                             | SMW         |
| 497 | Mechakoopa                        | SMW         |
| 498 | Mechakoopa (Stunned)              | SMW         |
| 499 | Big Steely                        | SMW         |
| 500 | Bowser                            | SMW         |
| 501 | Reznor                            | SMW         |
| 502 | Reznor's Fireball                 | SMW         |
| 503 | Iggy Koopa                        | SMW         |
| 504 | Iggy Koopa's Fireball             | SMW         |
| 505 | Morton Koopa Jr.                  | SMW         |
| 506 | Lemmy Koopa                       | SMW         |
| 507 | Ludwig von Koopa                  | SMW         |
| 508 | Wiggler                           | SMW         |
| 509 | Wiggler (Red)                     | SMW         |
| 510 | Urchin                            | SMW         |
| 511 | Torpedo Ted                       | SMW         |
| 512 | Torpedo Ted (Spawner)             | SMW         |
| 513 | Sumo Brother                      | SMW         |
| 514 | Lightning                         | SMW         |
| 515 | Lightning Fire                    | SMW         |
| 516 | Hopping Flame                     | SMW         |
| 517 | Hopping Flame (Trail)             | SMW         |
| 518 | Super Koopa                       | SMW         |
| 519 | Red Super Koopa                   | SMW         |
| 520 | Blue Super Koopa                  | SMW         |
| 521 | Cape Feather                      | SMW         |
| 522 | Skewer                            | SMW         |
| 523 | Pillar                            | SMW         |
| 524 | Haunted Hole                      | SMW         |
| 525 | Floating Mine                     | SMW         |
| 526 | Fishin' Lakitu                    | SMW         |
| 527 | Falling Spike                     | SMW         |
| 528 | Lil Sparky                        | SMW         |
| 529 | Hothead                           | SMW         |
| 530 | Monty Mole                        | SMW         |
| 531 | Parachute                         | SMW         |
| 532 | Wing                              | SMW         |
| 533 | Porcu-Puffer                      | SMW         |
| 534 | Rip Van Fish                      | SMW         |
| 535 | Gray P Switch                     | SMW         |
| 536 | Gray Coin                         | SMW         |
| 537 | Red Berry                         | SMW         |
| 538 | Pink Berry                        | SMW         |
| 539 | Green Berry                       | SMW         |
| 540 | Baby Yoshi                        | SMW         |
| 541 | Bouncing bough                    | SMW         |
| 542 | Diagonal platform                 | SMW         |
| 543 | Amazing Flyin' Hammer Brother     | SMW         |
| 544 | Hammer                            | SMW         |
| 545 | Swing Lift                        | SMW         |
| 546 | Swing Lift (Rotating)             | SMW         |
| 547 | Flip Fence Panel                  | SMW         |
| 548 | Climbing Koopa                    | SMW         |
| 549 | Red Climbing Koopa                | SMW         |
| 550 | Ball 'N' Chain                    | SMW         |
| 551 | Chain                             | SMW         |
| 552 | Big Bubble                        | SMW         |
| 553 | Boo Buddy Block                   | SMW         |
| 554 | Boo Buddy (Special)               | SMW         |
| 555 | Circling Boos                     | SMW         |
| 556 | Boo Snake                         | SMW         |
| 556 | Fuzzy                             | SMW         |
| 557 | Lava Bubble                       | SMW         |
| 558 | Diagonal Lava Bubble              | SMW         |
| 559 | Piranha Plant                     | SMW         |
| 560 | Jumping Piranha Plant             | SMW         |
| 561 | Jumping Fire Piranha Plant        | SMW         |
| 562 | Piranha Plant (Horizontal)        | SMW         |
| 563 | Jumping Piranha Plant (Horizontal)              | SMW         |
| 564 | Jumping Fire Piranha Plant (Horizontal)         | SMW         |
| 565 | Pipe Lakitu                       | SMW         |
| 566 | Chargin' Chuck                       | SMW         |
| 567 | Clappin' Chuck                       | SMW         |
| 568 | Passin' Chuck                        | SMW         |
| 569 | Lookout Chuck                        | SMW         |
| 570 | Confused Chuck                       | SMW         |
| 571 | Diggin' Chuck                        | SMW         |
| 572 | Bouncin' Chuck                       | SMW         |
| 573 | Whistlin' Chuck                      | SMW         |
| 574 | Football                             | SMW         |
| 575 | Rock                                 | SMW         |
| 576 | Baseball                             | SMW         |
| 577 | Blue Dry Bones                       | SMW         |
| 578 | Blue Para-Dry Bones                  | SMW         |
| 579 | Para-Dry Bones                       | SMW         |
| 580 | Bony Beetle                          | SMW         |
| 581 | Boo Crew                             | SMW         |
| 582 | Decoy                                | SMW         |
| 582 | Fireball (Enemy)                     | SMW         |
| 582 | Power Balloon                        | SMW         |
| 583 | Stretch Block                        | SMW         |
| 584 | Spotlight                            | SMW         |
| 585 | Bumper                               | SMW         |
| 586 | Baby Bumper                          | SMW         |
| 587 | Donut Block                          | SMW         |
| 588 | Giant P Switch                       | SMW         |
| 589 | Bob-Omb                              | SMW         |
| 590 | Bob-Omb (Stunned)                    | SMW         |

P.S: Some npcs may be deprecated in favour of new ones, for example plants.
P.P.S: Some npcs have functions only in overworld (worldmap).

**Comment by LoveBodhi** [comment]
Misc Section ideas by me:

- 591-593 Platforms (YI)
- 594 Gasty (YI)
- 595 Numbered Platform (YI)
- 596 Wild Piranha Plant
- 597 Wild Ptooie
- 598 Camera-based Generator

**Comment by vitzie629** [comment]
Idea:
599 Flying Poison Mushroom (MF, SMB1)
600 Beetroot Power-Up (MF, SMB1)
601 Jumping Lui (MF, SMB1)
602 Blue Cheep Cheep (MF, SMB1)
603 Yellow Spike Cheep Cheep (MF, SMB1)
604 Mega Mushroom (NSMB)
605 Mini Mushroom (NSMB)

**Comment by LoveBodhi** [comment]
Then:

- 606 Bumpty (YI)
- 607 Taptap (YI)
